cpg_infra/billing_aggregator/aggregate/utils.py

```diff
@@ -1151,3 +1150,20 @@ def infer_batch_namespace(batch: dict) -> str:
             return 'main'
 
     return default
+
+
+def format_as_string(data: list[dict[str, Any]]) -> str:
```

Comment:
Can you rename this to something bigquery related?
Maybe even doing this in a two step (convert from the [{key: key, value: value}] to {} so it's easier to work with, then to string as a later when we're about to send it to bigquery.

Reply:
Have renamed to reformat_bigqquery_labels and keep results to dict, make changes so when we assign to BQ record it gets converted to string."